PRD.md

```
@@ -1,47 +1,64 @@
+| **Phase 6** | code · external-repo | Publishing infrastructure — external platforms |
+| **Phase 7** | code · external-repo | Platform migration |
+| **Phase 8** | code · this-repo | Final cleanup |
+| **Phase 9** | code · external-repo | Upstream contributions |
+| **Phase 10** | content · external-repo | Content authority & off-site |
+| **Post-Migration** | content · this-repo | Article edits |
+| **Site Defects** | code · this-repo | Live-site rendering bugs |
+| **Documentation** | code · external-repo | READMEs, DocC, cross-repo docs |
+
 ### Dependency Chain
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)